packages/reactive-element/src/reactive-element.ts

```diff
@@ -219,21 +219,32 @@ type PropertyDeclarationMap = Map<PropertyKey, PropertyDeclaration>;
 type AttributeMap = Map<string, PropertyKey>;
 
 /**
- * Map of changed properties with old values. Takes an optional generic
- * interface corresponding to the declared element properties.
+ * A Map of property keys to values.
+ *
+ * Takes an optional type parameter T, which when specified as a non-any,
+ * non-unknown type, will make the Map more strongly-typed, associating the map
+ * keys with their corresponding value type on T.
+ *
+ * Use `PropertyValues<this>` when overriding ReactiveElement.update() and
+ * other lifecycle methods in order to get stronger type-checking on keys
+ * and values.
  */
```

Comment:
Is the jsdoc comment going to get associated with this type correctly, given that there's a `//` commen below it? I think if you can hover over it in vscode, it should be fine. Less certain that typedoc would also work.

Reply:
yeah, it works fine.